integration/tests/update/update_cluster_test.go

```diff
@@ -119,8 +121,7 @@ var _ = Describe("(Integration) Update addons", func() {
 			Expect(config).To(HaveExistingStack(fmt.Sprintf("eksctl-%s-nodegroup-%s", params.ClusterName, initNG)))
 		})
 
-		It(fmt.Sprintf("should upgrade the control plane to version %s", nextEKSVersion), func() {
-
+		It("should upgrade the control plane to the next version", func() {
```

Comment:
do we need this change? the original line matches the assertion for lines 134 and below

Reply:
We do. `nextEKSVersion` is not set here.